automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -160,16 +160,18 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
 
   }
 
-  "import a method config into a workspace from the method repo" ignore withWebDriver { implicit driver =>
+  "import a method config into a workspace from the method repo" in withWebDriver { implicit driver =>
     val user = UserPool.chooseProjectOwner
     implicit val authToken: AuthToken = user.makeAuthToken()
     withWorkspace(billingProject, "TestSpec_FireCloud_import_method_config_from_workspace") { workspaceName =>
-      api.importMetaData(billingProject, workspaceName, "entities", TestData.SingleParticipant.participantEntity)
-      withSignIn(user) { _ =>
-        val workspaceMethodConfigPage = new WorkspaceMethodConfigListPage(billingProject, workspaceName).open
-        val methodConfigDetailsPage = workspaceMethodConfigPage.importMethodConfigFromRepo(SimpleMethodConfig.configNamespace,
-          SimpleMethodConfig.configName, SimpleMethodConfig.snapshotId, methodConfigName)
-        //    methodConfigDetailsPage.editMethodConfig(inputs = Some(TestData.SimpleMethodConfig.inputs)) // not needed for config
+      withSignIn(user) { workspaceListPage =>
+        val methodConfigPage = workspaceListPage.enterWorkspace(billingProject, workspaceName).goToMethodConfigTab()
```

Comment:
I propose that we always click through the app instead of jumping straight to pages

Reply:
Some tests need to check direct access which is perfectly valid when people have bookmarks to things. We should absolutely have some tests for checking navigation to things. Given those, I don't think it matters much how other tests navigate.

Reply:
Internally, navigation works by setting the URL and letting the application rerender. There's no functional difference between clicking on a button/link or entering the URL.